### DISCUSSION TITLE
Can we please get a list of user-requested features and the development of them, or something similar?

### DISCUSSION THREAD

**Discussion by Shidouuu** [Tribler/tribler#5872] [Ideas] [answered]
I apologize in advance if there is already something like this.

There are a ton of basic features that are in other bittorrent clients, but not in Tribler. Something like a poll or list in which users can vote on which features they want implemented would help with this.

Here are just a few features that I feel should've been implemented by now:

- Low customization ability
This one isn't a huge deal but still somewhat annoying. Everything feels too large and clunky compared to other clients. There's no way to change the size of the rows, no way to adjust the size of the left pop-out menu, and if you close it everything else goes right to the edge of the screen; no margin. Additionally, you can't hide any of the columns, so you're stuck with viewing some useless information. 
- No torrent searches routed thru TOR
I feel this is strange considering what Tribler aims to be. The current search function has almost no use compared to getting a torrent from the web. If it were routed thru TOR it would make a lot more sense to use it.
- Torrents are either `downloading`, `seeding` or `stopped`
This is a problem because if you were to delete a torrent(s) outside Tribler it will begin downloading again. There should be an `error` state if the torrent data is not found, which requires that Tribler check for the data first, which it does not currently do.
- No option to run at system start
Again not a huge deal but I feel this is a pretty basic feature and it is annoying not to have it.

**Comment by ichorid** [comment, marked as answer]
Dear @Shidouuu , thank you for your interest in our project! As you can see, our GitHub Issues list is quite a mess! Also, at this moment the team is mostly taking their holiday vacation.

We will be *very* grateful if someone would sift through our 345 issues and 16 projects and assemble a list of user-requested features :wink:

**Comment by Shidouuu** [comment]
I wrote this for no good reason. I apologize.
